frontend/src/views/AllocationAgreements/_schema.jsx

```diff
@@ -211,13 +212,12 @@ export const allocationAgreementColDefs = (optionsData, errors) => [
     ),
     cellStyle: (params) => {
       const style = cellErrorStyle(params, errors)
-      const conditionalStyle =
-        params.data.fuelType === 'Other'
+      const conditionalStyle = params.data.unrecognized
           ? { backgroundColor: '#fff', borderColor: 'unset' }
           : { backgroundColor: '#f2f2f2' }
       return { ...style, ...conditionalStyle }
     },
-    editable: (params) => params.data.fuelType === 'Other',
```

Comment:
We have to leave this: params.data.fuelType === 'Other' 
Reason is that we need to allow input of new custom fuel_types if Other is selected.